velox/functions/lib/string/tests/StringImplTest.cpp

```diff
@@ -282,6 +282,19 @@ TEST_F(StringImplTest, stringToCodePoints) {
       "Invalid UTF-8 encoding in characters");
 }
 
+TEST_F(StringImplTest, findNthInstanceByteIndexFromStart) {
+  auto testValidInputAscii = [](const std::string& string,
+                                const std::string& substr,
+                                const int64_t instance,
+                                const int64_t expectedPosition) {
+    auto result = findNthInstanceByteIndexFromStart(
+        std::string_view(string), std::string_view(substr), instance);
+    ASSERT_EQ(result, expectedPosition);
+  };
+
+  testValidInputAscii("|||", "||", 2, 1L);
+}
+
```

Comment:
shall we add a unit test for strpos and strrpos

Reply:
Added.